services/src/main/java/org/keycloak/crypto/ServerECDSASignatureVerifierContext.java

```diff
@@ -0,0 +1,37 @@
+package org.keycloak.crypto;
+
+import org.keycloak.common.VerificationException;
+import org.keycloak.models.KeycloakSession;
+
+public class ServerECDSASignatureVerifierContext extends  AsymmetricSignatureVerifierContext {
+    public ServerECDSASignatureVerifierContext(KeycloakSession session, String kid, String algorithm) throws VerificationException {
+        super(getKey(session, kid, algorithm));
+    }
+
+    public ServerECDSASignatureVerifierContext(KeyWrapper key) {
+        super(key);
+    }
+
+    private static KeyWrapper getKey(KeycloakSession session, String kid, String algorithm) throws VerificationException {
```

Comment:
Perhaps `ServerAsymmetricSignatureVerifierContext.getKey` could be made package-protected and called instead of introducing this new method.